src/Nethermind/Nethermind.JsonRpc/Modules/Trace/TraceRpcModule.cs

```diff
@@ -210,7 +210,7 @@ public ResultWrapper<ParityTxTraceFromStore[]> trace_transaction(Keccak txHash)
 
             Block block = blockSearch.Object;
 
-            ParityLikeTxTrace txTrace = TraceTx(block, txHash, ParityTraceTypes.Trace | ParityTraceTypes.Rewards);
+            ParityLikeTxTrace txTrace = TraceTx(block, txHash, ParityTraceTypes.Trace);
```

Comment:
hmmm, ok, this is interesting -> Lukasz definitely uses rewards traces

Reply:
it looks like comparison of conditions is broken somewhere (== instead of &=) -> I do not think that this fix is correct (gut feeling)

Reply:
Ok, It looks like OE and Erigon don't return rewards for trace_transaction. I can fix it to have calls & rewards.